drake/systems/plants/constraint/RigidBodyConstraint.h

```diff
@@ -427,7 +432,9 @@ class DRAKERIGIDBODYCONSTRAINT_EXPORT GazeConstraint : public SingleTimeKinemati
     virtual ~GazeConstraint(void){};
 
 public:
+#ifndef SWIG
```

Comment:
Actually, this turns out to be harder than you'd think, because in C++ you can't create a macro which contains an `#ifdef` directive. I think unless this is something we want to spend significant time on, we might just have to live with it.